controller/controller.go

```diff
@@ -158,6 +158,8 @@ func (f *Controller) DeleteFunc(obj interface{}) {
 	f.mutex.Lock()
 	defer f.mutex.Unlock()
 
+	fmt.Printf("1\n")
```

Comment:
This should be removed.

Reply:
Yeah totally. Sorry. This is still WIP. Requested reviews for awareness. Will get back to you later.